source/material_model/interface.cc

```diff
@@ -440,221 +440,220 @@ namespace aspect
       }
 
 
-      namespace
```

Comment:
Yea, let's please stick with the anonymous namespace. If you get an error message, let's take a look at the error and fix it, rather than work around it.